starter_scripts/exaitde_within_docker_container.sh

```diff
@@ -91,13 +92,14 @@ function create_env_file_debug_protected() {
 }
 
 old_umask=$(umask)
-umask 277
+umask 077
```

Comment:
Do I understood correctly that only one file with the new `umask` created?
If so, creating the file and then doing a `chmod` would not only save an extra loc (instruction)
it would be "saver" (undoing the umask can't be forgotten).

Reply:
1. We could `touch` the file, then `chmod` it with user r/w , then write the secrets, and afterwards set it readonly. Still would need the extra step of one more `chmod`.
2. I thought that too, but `umask` is valid per process, so if the bash script ends, it automatically resets to the parent's process umask.

Reply:
I changed the implementation now and use only `chmod`.